workflow/scripts/copy_files.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+import pandas as pd
+import os
+import yaml
+
+try:
+    output = snakemake.params.output
+except NameError:
+    with open('config/config.yaml', 'r') as f:
+        config = yaml.safe_load(f)
+    output = config['output']
+
+
+# Copy all files relevant for the clinic to the output folder
+def copy_files():
+    df = pd.read_csv(f'{output}/samtools/minimum_error_rates.csv')
+    for _, row in df.iterrows():
+        read_id = row['read_id']
+        ref = row['ref']
+
+        consensus_in = f'{output}/consensus/medaka/{read_id}.{ref}/consensus.fasta'
+        consensus_out = f'{output}/clinic/{read_id}.{ref}.fa'
+
+        os.makedirs(f'{output}/clinic', exist_ok=True)
+
+        # Change headers of consensus files and write to output folder
+        with open(consensus_in, 'r') as in_fasta, open(consensus_out, 'w') as out_fa:
+            for line in in_fasta:
+                if line.startswith('>'):
+                    out_fa.write(f'>{read_id}.{ref}\n')
```

Comment:
`print(f'>{read_id}.{ref}', file=out_fa)` is an alternative if you dont want to add newlines all the time.